user-guide/Advanced_Modules/User Defined APIs/Tutorial/UD_APIs_Elements_in_alarm_tutorial.md

```diff
@@ -0,0 +1,120 @@
+---
+uid: UD_APIs_Elements_in_alarm_tutorial
+---
+
+# Elements in alarm tutorial
```

Comment:
I would change the title to something like 'Create an API to retrieve Elements in alarm'